src/utilities/general.h

```diff
@@ -47,5 +47,17 @@ typedef double                 hypre_double;
 #define hypre_pow2(i)  ( 1 << (i) )
 #endif
 
+#include <HYPRE_config.h>
```

Comment:
You shouldn't need this line since this is only used to help build `_hypre_utilities.h` which should already have it by way of `HYPRE_utilities.h`.

Reply:
Thanks @rfalgout, this makes sense, but I don't see general.h including any header files. It is used directly in the Makefile as part of the HEADERS target (which includes _hypre_utilities.h).

Reply:
You are right that a bunch of header files could be deleted from the `Makefile` dependency list since they are only used to generate `_hypre_utilities.h`.  You still should be able to delete this line though, I think.